packages/client/src/types/sigstore/index.ts

```diff
@@ -13,26 +13,54 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import {
+import { Bundle, HashAlgorithm } from '@sigstore/protobuf-specs';
+import { encoding as enc, pem } from '../../util';
+import { SignatureMaterial } from '../signature';
+import { ValidBundle, assertValidBundle } from './validate';
+
+import type {
   ArtifactVerificationOptions,
-  Bundle,
   Envelope,
-  HashAlgorithm,
   TimestampVerificationData,
   TransparencyLogEntry,
   VerificationMaterial,
 } from '@sigstore/protobuf-specs';
-import { encoding as enc, pem } from '../../util';
-import { x509Certificate } from '../../x509/cert';
-import { WithRequired } from '../utility';
-import { ValidBundle, assertValidBundle } from './validate';
-
 import type { Entry, ProposedEntry } from '../../external/rekor';
-import type { SignatureMaterial } from '../signature';
+import type { WithRequired } from '../utility';
+import type { SerializedBundle } from './serialized';
 
-export * from '@sigstore/protobuf-specs';
-export * from './serialized';
-export * from './validate';
+// Enums from protobuf-specs
+// TODO: Move Envelope to "type" export once @sigstore/sign is a thing
+export {
+  Envelope,
+  HashAlgorithm,
+  PublicKeyDetails,
+  SubjectAlternativeNameType,
+} from '@sigstore/protobuf-specs';
+// Types from protobuf-specs
+export type {
+  ArtifactVerificationOptions,
+  ArtifactVerificationOptions_CtlogOptions,
+  ArtifactVerificationOptions_TlogOptions,
+  CertificateAuthority,
+  CertificateIdentities,
+  CertificateIdentity,
+  MessageSignature,
+  ObjectIdentifierValuePair,
+  PublicKey,
+  PublicKeyIdentifier,
+  RFC3161SignedTimestamp,
+  Signature,
+  SubjectAlternativeName,
+  TimestampVerificationData,
+  TransparencyLogEntry,
+  TransparencyLogInstance,
+  TrustedRoot,
+  X509Certificate,
+  X509CertificateChain,
+} from '@sigstore/protobuf-specs';
```

Comment:
Instead of exporting EVERYTHING out of `@sigstore/protobuf-specs` I'm trying to be a bit more targeted and export only the types we're actually using. This will help us keep track of what things we're using and help inform what sorts of changes in the upstream package may be breaking.